db/data_migration/20170814083356_republish_world_location_news_pages.rb

```diff
@@ -1,4 +1,4 @@
 WorldLocation.where(world_location_type_id: 1).each do |world_location|
   puts "Republishing news page for #{world_location.name}"
-  WorldLocationNewsPageWorker.new.perform(world_location.id)
+  WorldLocationNewsWorker.new.perform(world_location.id)
```

Comment:
I don't know whether we should change this in historic migrations or not?

It does raise a question about how long migrations should exist for after running in production, before we delete them 🤔.

Reply:
Yeah I wasn't sure about this so had a chat with Ben about this last week, we came down on the side of they should be changed since if we needed to run all migrations from scratch it wouldn't work.